README.md

```diff
@@ -4,57 +4,62 @@ This Django project template is designed for creating Wagtail builds quickly, in
 
 ## Getting Started
 
-1. **Create a Virtual Environment**: Set up a virtual environment to isolate your project dependencies.
-
-    ```bash
-    pyenv virtualenv 3.8 myproject
-    pyenv activate myproject
+1. **Check that you have an appropriate version of Python 3**  You want to make sure that you have a [compatible version](https://docs.wagtail.org/en/stable/releases/upgrading.html#compatible-django-python-versions) installed:
+
+    ```sh
+    python --version
+    # Or:
+    python3 --version
+    # **On Windows** (cmd.exe, with the Python Launcher for Windows):
+    py --version
     ```
 
-2. **Install Wagtail**: Install the Wagtail CMS package using pip.
+2. **Create a Virtual Environment**: Set up a virtual environment to isolate your project dependencies. These instructions are for GNU/Linux or MacOS, but there are [other operating systems in the Wagtail docs](https://docs.wagtail.org/en/stable/getting_started/tutorial.html#create-and-activate-a-virtual-environment).
 
     ```bash
-    pip install wagtail
+    python -m venv myproject/env
+    source myproject/env/bin/activate
     ```
 
-3. **Initialize Project**: Use the Django `startproject` command to create a new project based on the Wagtail Starter Kit template.
+4. **Navigate to Project Directory**: Move into the newly created project directory.
 
     ```bash
-    wagtail start myproject --template=<<path or git url>>
+    cd myproject
     ```
 
-4. **Navigate to Project Directory**: Move into the newly created project directory.
+5. **Install Wagtail**: Install the Wagtail CMS package using pip.
 
     ```bash
-    cd myproject
+    pip install wagtail
     ```
 
-5. **Install Project Dependencies**: Install the project's dependencies into a virtual environment.
+6. **Initialize Project**: Use the Django `startproject` command to create a new project based on the Wagtail Starter Kit template.
 
     ```bash
-    python -m venv venv
+    wagtail start --template=https://github.com/torchbox/wagtail-news-template/archive/refs/heads/main.zip myproject .
+    ```
 
-    source venv/bin/activate  # For Linux / macOS
-    venv/bin/Activate.ps1  # For Windows
+7. **Install Project Dependencies**: Install the project's dependencies into a virtual environment.
 
+    ```bash
     pip install -r requirements.txt
     ```
 
 All commands from now on should be run from inside the virtual environment.
 
-6. **Load Dummy Data**: Optionally load in some dummy data, to populate the site with some content.
+7. **Load Dummy Data**: Load in some dummy data to populate the site with some content.
```

Comment:
Nit: "7" is duplicated here - should be 8.

Reply:
Great catch. Thank you!